README.md

```diff
@@ -86,19 +58,73 @@ Or you can use this instead if your build pipeline supports ES6 imports:
 import { fabric } from "fabric";
 ```
 
+NOTE: If you are using Fabric.js in a Node.js script, you will depend from [node-canvas](https://github.com/Automattic/node-canvas).`node-canvas` is an HTML canvas replacement that works on top of native libraries. Please [follow the instructions](https://github.com/Automattic/node-canvas#compiling) to get it up and running.
+
 NOTE: es6 imports won't work in browser or with bundlers which expect es6 module like vite. Use commonjs syntax instead.
 
-See [the example section](#examples-of-use) for usage examples.
+### Usage
+
+#### Plain HTML
+
+```html
+<canvas id="canvas" width="300" height="300"></canvas>
+
+<!-- Get latest version: https://cdnjs.com/libraries/fabric.js -->
+<script src="lib/fabric.js"></script> 
+<script>
+    var canvas = new fabric.Canvas('canvas');
+
+    var rect = new fabric.Rect({
+        top : 100,
+        left : 100,
+        width : 60,
+        height : 70,
+        fill : 'red'
+    });
+    canvas.add(rect);
+</script>
+```
+
+#### ReactJS
+
+```tsx
+import React, {useEffect, useRef} from 'react'
+import {fabric} from 'fabric'
+
+const FabricJSCanvas = () => {
+  const canvasEl = useRef(null)
+  useEffect(() => {
+    const canvas = new fabric.Canvas(canvasEl.current);
+    const rect = new fabric.Rect({
+            top : 100,
+            left : 100,
+            width : 60,
+            height : 70,
+            fill : 'red'
+    });
+    
+    canvas.add(rect);
+    return () => {
+      canvas.dispose()
+    }
+  }, []);
+  
+  return (<canvas width="300" height="300" ref={canvasEl}/>)
+});
+
+export default FabricJSCanvas;
+```
+
+NOTE: Fabric.js requires a `window` object.
 
-<h3 id="fabric-building">Building</h3>
 
-1. [Install Node.js](https://github.com/joyent/node/wiki/Installation)
+<h3 id="fabric-building">Building</h3>
```

Comment:
I vote to get rid of this entire section for 3 main reasons:
1. too esoteric
2. there's a new cli for fabric that is very simple to use - PRs should be made to make it better instead of this bloated part about building options.
3. The old build script is at it's end of life and will soon die out.

Reply:
I also vote for removing this part or move it to another document and just leave the link to it somewhere.